Cargo.toml

```diff
@@ -1,22 +1,23 @@
 [package]
-name = "firec"
 description = "Rust API to interact with firecracker"
-version = "0.1.0"
 edition = "2021"
+name = "firec"
+version = "0.1.0"
```

Comment:
Unrelated change, please remove.

Reply:
put into separate commit

Reply:
I don't get this part though. I'd expect the name to be the first thing listed, before description.

Reply:
it's automatically sorted just as blocks of mod's or use's

Reply:
oh that's not rust fmt work but my formatter for toml
so this is definitely debatable

Reply:
yeah, rustfmt only handles rust files (for now at least). Your toml formatter probably doesn't know about Cargo but works in a generic way.

I checked the Cargo.toml of top most downloaded crates on crates.io and they all have name and version as the first thing. I hope you can agree this makes sense.